lib/booking/ui/calendar.dart

```diff
@@ -91,124 +92,117 @@ class Calendar extends HookConsumerWidget {
       }
     }
 
-    return LayoutBuilder(
```

Comment:
I removed the LayoutBuilder widget in commit [473da4d](https://github.com/aeecleclair/Titan/pull/176/commits/473da4d246b32fc276d73e86c0d988618680be7e) because it did not seem useful to me. The SfCalendar chooses by default to take the maximum of avaible space.